web/src/utils/state/atoms.ts

```diff
@@ -29,8 +29,19 @@ export const spatialAggregateAtom = atomWithStorage(
 );
 export const productionConsumptionAtom = atomWithStorage('mode', Mode.CONSUMPTION);
 
-export const solarLayerEnabledAtom = atomWithStorage('solar', ToggleOptions.OFF);
+export const solarLayerAtom = atomWithStorage('solar', ToggleOptions.OFF);
+export const isSolarLayerEnabledAtom = atom(
+  (get) =>
+    get(solarLayerAtom) === ToggleOptions.ON &&
+    get(selectedDatetimeIndexAtom).index === 24
```

Comment:
We need to add a check for `isHourly` (from #6980), as it is currently possible to make the solar layer show on daily view (on day 25) 😬 

![Screenshot 2024-07-16 at 14 40 24](https://github.com/user-attachments/assets/0aef23a8-d619-47a4-89bc-6e6a5cbc90c7)

Reply:
Should be fixed now